recipe/meta.yaml

```diff
@@ -10,12 +10,12 @@ source:
   sha256: 25949c8db62684bc9ce7ba9ee26501a75709a096845644e13993ff6aee5d71d7
 
 build:
-  number: 0
-  noarch: generic
+  number: 1
 
 requirements:
   build:
-    - unzip
+    - unzip  # [unix]
```

Comment:
Can osx be added to this? Does this now not work for osx?

Reply:
yes, it should still be working the same way as before on both OSX and Linux. the `# [unix]` selector is just saying that unzip is only required if the platform is either macOS or Linux.